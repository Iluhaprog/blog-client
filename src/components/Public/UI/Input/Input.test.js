import {render} from '@testing-library/react';
import {Input} from './index';

test('Should render Input component', () => {
  const {asFragment} = render(
    <Input
      value={'Ok'}
      placeholder={'Test'}
      onChange={() => {}}
    />);
  expect(asFragment()).toMatchSnapshot();
});
