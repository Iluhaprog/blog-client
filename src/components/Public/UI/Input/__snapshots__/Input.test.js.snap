// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Input component 1`] = `
<DocumentFragment>
  <input
    class="sc-bdfBwQ izwldC"
    placeholder="Test"
    value="Ok"
  />
</DocumentFragment>
`;
